docs/reference/metadata.md

```diff
@@ -0,0 +1,1273 @@
+- ``place`` (0-1) - free text, used to describe a geographical location.
+- ``description`` (0-1) - free text, used for any extra information related to the location.
+
+Example:
+
+```json
+{
+    "locations": [{
+      "geometry": {
+        "type": "Point",
+        "coordinates": [6.05, 46.23333]
+      },
+      "identifiers": {
+        "geonames": "2661235",
+        "tgn": "http://vocab.getty.edu/tgn/8703679"
+      },
+      "place": "CERN",
+      "description": "Invenio birth place."
+    }]
+}
+```
+
+### Funding references (0-n)
+
+!!! warning "Work in progress"
+    This field may change after work done on controlled vocabulary management is
+    completed during December 2020.
+
+Information about financial support (funding) for the resource being registered.
+
+This field is compatible with *19. Funding Reference* in DataCite.
+
+The funder subfield is intended to be linked to a customizable vocabulary from Open Funder Registry or ROR. The award subfields is intended to be either linked to a customizable vocabulary sourced from the OpenAIRE grant database, or be specified explicitly to allow linking to grants not provided by the grant database.
+
+Subfields:
+
+- ``funder`` (1) - The organisation of the funding provider.
+- ``award`` (0-1) - The award (grant) sponsored by the funder.
+
+Funder subfields:
+
+- ``name`` - The name of the funder.
+- ``identifer`` -  An unique identifier for the funder.
+- ``scheme`` - The scheme of the identifier.
+
+Supported schemes:
+
+- CrossRef Funder ID
+- GRID
+- ISNI
+- ROR
+- Wikidata
+
+Award subfields:
+
+- ``title`` (0-1) - The title of the award
+- ``number`` (0-1) - The code assigned by the funder to a sponsored award (grant).
+- ``identifer`` (0-1) - An unique identifier for the funder.
+- ``scheme`` (0-1) - The scheme of the identifier.
+
+Example:
+
+```json
+{
+  "funding": [{
+      "funder": {
+        "name": "European Commission",
+        "identifier": "00k4n6c32",
+        "scheme": "ror"
+      },
+      "award": {
+        "title": "OpenAIRE",
+        "number": "246686",
+        "identifier": ".../246686",
+        "scheme": "openaire"
+      }
+  }]
+}
+```
+
+### References (0-n)
+
+!!! warning "Work in progress"
+    This field may be removed prior to the first stable release.
+
+A list of reference strings
+
+Subfields:
+
+- ``reference`` (1) - free text, the full reference string
+- ``identifier`` (0-1) - the identifer if known.
+- ``scheme`` (0-1) - the scheme of the identifier.
+
+Example:
+
+```json
+{
+  "references": [{
+      "reference": "Nielsen et al,..",
+      "identifier": "10.1234/foo.bar",
+      "scheme": "doi"
+  }]
+}
+```
+
+## Extensions
+
+InvenioRDM supports extending records with custom metadata per instance. The
```

Comment:
Shall we put a `!!! warning` "This is currently not working and is subtle to changes."""

Reply:
Yes. subtle -> subject